src/transformers/models/audio_spectrogram_transformer/feature_extraction_audio_spectrogram_transformer.py

```diff
@@ -182,6 +191,12 @@ def __call__(
                 - `'tf'`: Return TensorFlow `tf.constant` objects.
                 - `'pt'`: Return PyTorch `torch.Tensor` objects.
                 - `'np'`: Return Numpy `np.ndarray` objects.
+            timem (`int`, *optional*):
+                The maximum possible length of the mask in the time domain.
+            freqm (`int`, *optional*):
+                The maximum possible length of the mask in the frequency domain.
+            add_noise (`bool`, *optional*):
+                Whether or not to add noise to the input.
```

Comment:
Let's find more explicit name and descriptions here!

Something like `time_mask_length` and `frequency_mask_length` and descriptions that explain that if they're set, it'll apply a mask over the time / frequency training. And that it's useful for training ?

Reply:
And specify that torchaudio is required